src/main/java/io/github/moulberry/notenoughupdates/profileviewer/BasicPage.java

```diff
@@ -672,6 +623,12 @@ public String getSkinType() {
 					getInstance().renderBar(x, y + 6, 80, level.level % 1);
 				}
 
+				if (mouseX >= guiLeft + 128 && mouseX <= guiLeft + 216) {
```

Comment:
There should be a utils method for this. Seen somewhere around crafting recipe display gui.

Reply:
Not relevant enough ig